yoyodyne/train.py

```diff
@@ -165,6 +166,20 @@ def get_model_from_argparse_args(
         models.BaseEncoderDecoder.
     """
     model_cls = models.get_model_cls(args.arch)
+    if (
+        args.arch
+        in [
```

Comment:
Considering adding the following (or something like this):

`# TODO(#156): add callback interface to check this.`